docs/openapi.yml

```diff
@@ -102,7 +102,7 @@ paths:
                   example: "https://github.com/microclimate-dev2ops/nodeExpressTemplate"
       responses:
         200:
-          description: Project creation successful
+          description: Project creation request sent successfully
```

Comment:
should this be request received successfully ?

Reply:
That's a better way of putting it, and I've gone one step further to 'Project creation request received and processed' because we've done the work and the user needs to check the response to see if it was successful or not